archivebox/extractors/headers.py

```diff
@@ -0,0 +1,74 @@
+__package__ = 'archivebox.extractors'
+
+from pathlib import Path
+
+from typing import Optional
+
+from ..index.schema import Link, ArchiveResult, ArchiveOutput
+from ..system import atomic_write
+from ..util import (
+    enforce_types,
+    get_headers,
+    is_static_file,
+)
+from ..config import (
+    TIMEOUT,
+    CURL_BINARY,
+    CURL_USER_AGENT,
+    CURL_VERSION,
+    CHECK_SSL_VALIDITY,
+)
+from ..logging_util import TimedProgress
+
+@enforce_types
+def should_save_headers(link: Link, out_dir: Optional[str]=None) -> bool:
+    out_dir = out_dir or link.link_dir
+    if is_static_file(link.url):
```

Comment:
shouldn't it still save the headers even if it's a static file?

Reply:
Yes, I will remove that condition.